web/tailwind.config.js

```diff
@@ -15,6 +15,7 @@ const config = {
     },
     extend: {
       colors: {
+        white: '#FFFFFF',
```

Comment:
I am *very* certain that this already exists

Reply:
Yes, I removed it